modules/terraform/show.go

```diff
@@ -28,3 +28,21 @@ func ShowE(t testing.TestingT, options *Options) (string, error) {
 	}
 	return RunTerraformCommandAndGetStdoutE(t, options, args...)
 }
+
+func ShowWithStruct(t testing.TestingT, options *Options) (*PlanStruct) {
```

Comment:
AFAIK think parentheses can be removed since it is single value returned

Reply:
Yes, probably forgot to format.